data/presets/man_made/ski_jump.json

```
@@ -0,0 +1,25 @@
+{
+    "icon": "temaki-ski_jumping",
+    "fields": [],
+    "moreFields": [
+        "building"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
+        "jump",
+        "ski jump",
+        "in-run",
+        "inrun",
+        "ski jump structure",
+        "ski jumping hill structure",
+        "ski jump in-run",
+        "ski jumping hill in-run tower"
+    ],
+    "tags": {
+        "man_made": "ski_jump"
+    },
```

would it make sense to add `sport=ski_jumping` to these features?

```suggestion
    "tags": {
        "man_made": "ski_jump"
    },
    "addTags": {
        "man_made": "ski_jump",
        "sport": "ski_jumping"
    },
```

---

at second glance, I guess it that tag would be mostly superfluous (as it can be considered implied from the primary tag).